### PR TITLE
Fix undefined URLError in wcsaxes test

### PR DESCRIPTION
**Pull request by saimn**
Seen in https://travis-ci.org/astropy/astropy/jobs/221723330#L2633 
cc @astrofrog

### CODE REVIEW

**Comment by MSeifert04** [comment]
test failures seem real. Any idea what caused these `no module extern.six.moves` exceptions?

File: astropy/visualization/wcsaxes/tests/datasets.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import time
 
+from ...extern.six.moves import urllib
```
Comment (MSeifert04) [inline astropy/visualization/wcsaxes/tests/datasets.py]: missing one `.` if I'm correct.

Comment (bsipocz) [inline astropy/visualization/wcsaxes/tests/datasets.py]: That sounds right

**Comment by bsipocz** [comment]
@saimn - Thanks for spotting this, apparently there were no URLErrors when #5932 was tested...

File: astropy/visualization/wcsaxes/tests/datasets.py (hunk 1 above)
Comment (saimn) [inline astropy/visualization/wcsaxes/tests/datasets.py]: Oh yes, silly me, I pushed that in a hurry before leaving and I didn't check. Thanks for noticing this !

**Comment by saimn** [comment]
@bsipocz - Yep, it seems these errors have been more frequent recently (issue with `data.astropy.org` ?)

**Comment by bsipocz** [comment]
@saimn - ``data.astropy.org`` seemed to be running when I've checked it in the browser this afternoon (but travis was failing at the same time). I'm not sure what's wrong with it, maybe someone with actual access to it could check it more thoroughly. @pllim @astrofrog @eteq?

**Comment by pllim** [comment]
@bsipocz , @jhunkeler maintains the backend access.

**Comment by jhunkeler** [comment]
According to our logs, no requests have failed to reach the web server. I'm guessing it's a networking issuing somewhere between Travis and STScI, but not STScI's server.

**Comment by bsipocz** [comment]
@jhunkeler - Thanks for the confirmation. Since this is still happening (e.g. with the latest build here), I'm opening a new issue as a reminder and merging this.

**Pull request merged**